Samples/OLED1Xplained/OLED1.lpr

```diff
@@ -0,0 +1,152 @@
+program OLED1;
+{
+  This file is part of Pascal Microcontroller Board Framework (MBF)
+  Copyright (c) 2015 -  Michael Ring
+  based on Pascal eXtended Library (PXL)
+  Copyright (c) 2000 - 2015  Yuriy Kotsarenko
+
+  This program is free software: you can redistribute it and/or modify it under the terms of the FPC modified GNU
+  Library General Public License for more
+
+  This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the FPC modified GNU Library General Public
+  License for more details.
+}
+
+{< MBED application shield demo program. }
+
+{$INCLUDE MBF.Config.inc}
+
+{.$define DEBUG}
+
+uses
+  heapmgr, // this is needed for SAMD10, but keep it in place for the others also !!!!
+  MBF.BitHelpers,
+  MBF.__CONTROLLERTYPE__.SystemCore,
+  MBF.__CONTROLLERTYPE__.GPIO,
+  MBF.__CONTROLLERTYPE__.SPI,
+  MBF.Displays.OLED1Xplained;
+
+const
+  {$if defined(has_samd20_xplained_pro) or defined(has_samd21_xplained_pro) or defined(has_samc21_xplained_pro)}
+    PinLED     = LED0;
```

Comment:
Most of those definitions will (hopefully) get opsolete when SPI Code is completely implemented.

Reply:
Hope so too !